dev/src/setup/setup.ts

```diff
@@ -108,7 +109,7 @@ export async function setup() {
         const env = new Environment(pair, defaultConfig())
         await env.isReady()
 
-        const dappStakeDefault = (await env.contractInterface.getDappStakeDefault()).muln(2)
+        const dappStakeDefault = (await env.contractInterface['dappStakeThreshold']()).muln(2)
```

Comment:
The function `getDappStakeDefault` no longer exists as we're now using typechain-polkadot to generate the contract interface for functions. `dapp_stake_threshold` is a storage field in the contract and I've added a getter by the same name to the contract:

https://github.com/prosopo/captcha/blob/007cde247e94e2e79f95e5c832abc856ae9c7213/packages/contract/src/contract/interface.ts#L108-L120

I can't use `env.contractInterface.dappStakeThreshold` because TS doesn't know that this function exists as its created after compile time. In future, we'll add some fancy trick to make TS aware of storage functions.